base.py

```diff
@@ -498,10 +498,21 @@ def update_time(scene):
 class MBDynReadLog(bpy.types.Operator):
     """ Imports MBDyn nodes and elements by parsing the .log file """
     bl_idname = "animate.read_mbdyn_log_file"
-    bl_label = "MBDyn .log file parsing"
```

Comment:
No I'd rather add the Enum property to the ones shown in the `MBDynImportPanel`. It should have a telling name and description, obviously.